theme/src/components/skip-link.js

```diff
@@ -0,0 +1,38 @@
+import {Link} from '@primer/components'
+import styled from 'styled-components'
+import React from 'react'
+
+function SkipLinkBase({...rest}) {
+  return (
+    <Link
+      {...rest}
+      backgroundColor="blue.6"
+      color="white"
+      p={3}
+      href="#skip-nav"
+      fontSize={14}
```

Comment:
I believe fontSize `1` maps to `14px` in our theme. 
```suggestion
      fontSize={1}
```